packages/ubuntu_desktop_installer/lib/app.dart

```diff
@@ -54,6 +55,7 @@ class UbuntuDesktopInstallerApp extends StatelessWidget {
           Routes.allocateDiskSpace: (context) => AllocateDiskSpacePage(),
           Routes.writeChangesToDisk: (context) => WriteChangesToDiskPage(),
           Routes.chooseYourLook: (context) => ChooseYourLookPage(),
+          Routes.whoAreYouPage: (context) => WhoAreYouPage()
```

Comment:
could you follow the `static Widget WhoAreYouPage.create()` pattern shown in the _After_ section of #147, please?

Reply:
Okay will do. How about defining an interface and let pages implement it? At least this is what I would do in Java :)

Reply:
The "interface" is https://api.flutter.dev/flutter/widgets/WidgetBuilder.html ;)

Reply:
> could you follow the `static Widget WhoAreYouPage.create()` pattern shown in the _After_ section of #147, please?

Looked to your example and I think this would be too much for this PR for now. Or, if you can do it "on-the-fly" of your proposal implementation feel free to commit into this branch :)